Auth.php

```diff
@@ -32,6 +37,7 @@ public function __construct(Model $userModel = null)
         }
 
         $this->userModel = $userModel;
+        $this->fallbackAuth = new \Piwik\Plugins\Login\Auth();
```

Comment:
Yeah, but I thought it was ugly to have two UserModel properties in the class. Personal preference, I could be convinced otherwise